embedchain/bots/slack.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import logging
+import os
+import signal
+import sys
+
+from dotenv import load_dotenv
+from flask import Flask, request
+from slack_sdk import WebClient
+
+from embedchain import App
+
+from .base import BaseBot
+
+load_dotenv()
```

Comment:
Do we need this? Probably not since there is only one env variable needed here.

Reply:
I think it's good to make sure .env files are loaded, in case the variables from there haven't been loaded into the environment yet.

But this happens during app init, so yes, I think I can safely remove it.